config/config.local.sample.neon

```diff
@@ -46,6 +46,7 @@ parameters:
 	imgproxy:
 		key: "0aa4c34cb6636fb8d4deacd150b1c7b4"
 		salt: "20030cf6ef8fc43168d77a7c05f4cd31"
+		projectId: "change-this-to-name-or-hash-of-this-project"
```

Comment:
tohle nikdo nezmění a je to dlouhé a bude nám to plnit logy, dal bych to jako
`projectId: "" # change this to name of this project`
s tím, že pak přidáš do imgproxy kontrolu, že to není prázdné